starctl

```diff
@@ -155,6 +155,10 @@ if [[ ${#STARGATE_ARGS[@]} -eq 0 ]]; then
     if [[ ! -z "$DISABLE_MBEAN_REGISTRATION" ]]; then
         STARGATE_ARGS+=("--disable-mbean-registration")
     fi
+
+    if [[ ! -z "$CASSANDRA_CONFIG_PATH" ]]; then
```

Comment:
@marksurnin since supporting cassandra.yaml is being treated as an experimental feature can you please remove this and the associated reference in Starter.java?